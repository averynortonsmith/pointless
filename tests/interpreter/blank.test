
output = [a, b, c] |> println

(_, _) = (1, 2)
(_, a) = ("asdf", 2)
(b, c) = ("asdf", 2)