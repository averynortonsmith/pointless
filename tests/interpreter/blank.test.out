[2, "asdf", 2]
